app/models/concerns/orchestration.rb

```diff
@@ -82,14 +90,26 @@ def record_conflicts
     @record_conflicts ||= []
   end
 
+  def skip_orchestration!
+    @skip_orchestration = true
+  end
+
+  def enable_orchestration!
+    @skip_orchestration = false
+  end
+
+  def skip_orchestration?
+    @skip_orchestration ||= Rails.env == "test"
```

Comment:
It has two reasons - first, the original code was written with `Rails.env == 'test'` inside the queuing mechanism so I just refactored it out to a method and second, it makes sense to force-stop the orchestration in tests scenarios (since it does way too much side effects) and enable it only if it is specified explicitly.
Take a look at `Host::Base` model - it enables the orchestration back after it's done. I could write code in such way that it would restore the state to the previous one, but it would be complicating the production code for thee sake of testing. I prefer general simplicity of the code, and a bit of complication in specific places that need orchestration in test environment.